pages/profile/read/views/profile.jsx

```diff
@@ -0,0 +1,228 @@
+import React, { Fragment } from 'react';
+import isEmpty from 'lodash/isEmpty';
+import format from 'date-fns/format';
+import { defineValue } from '../../../common/formatters';
+import { Snippet, Link } from '@asl/components';
+import PilApply from './pil-apply';
+import RoleApply from './role-apply';
+import PplApply from './ppl-apply';
+import { dateFormat } from '../../../../constants';
+
+class Profile extends React.Component {
+  render() {
+    const { id: estId } = this.props.establishment;
+
+    const {
+      pil,
+      address,
+      postcode,
+      telephone,
+      email,
+      roles,
+      projects,
+      establishments,
+      id
+    } = this.props.profile;
+
+    const allowedActions = this.props.allowedActions;
+    const title = this.props.title;
+    const activeProjects = projects.filter(
+      ({ establishmentId, status }) =>
+        status === 'active' && establishmentId === estId
+    );
+    const estRoles = roles.filter(
+      ({ establishmentId }) => establishmentId === estId
+    );
+
+    const profileRole = establishments.find(est => est.id === estId).role;
+
+    return (
+      <Fragment>
+        <div className='content'>
```

Comment:
this appears to be either a redundant `<div>`. If the `div.content` is needed for layout it should be in the containing component (either the page for the profile page or the dashboard element)

Reply:
done

Reply:
done